### ISSUE TITLE
How to find service URL for proactive messages?

### ISSUE THREAD

**Issue by DonHarishAxe** [MicrosoftDocs/bot-docs#2219]
Hi, my bot will only be sending proactive messages without any user input. I've configured a way to find out the chats that it needs to post to, and we can assume that the bot is already installed in those chats. How do I know the service URL for these chats? 

For chats in different tenants, the region changes in the URL https://smba.trafficmanager.net/<region>. Is there a reliable mapping from tenant region to the one in URL that is documented anywhere?

**Comment by JonathanFingold** [comment]
On most channels, the user is expected to interact with the bot directly at least once before the bot can send proactive messages to the user. Which channel or channels are you designing your bot for?

**Comment by DonHarishAxe** [comment]
I'm designing my bot for Microsoft Teams

**Comment by JonathanFingold** [comment]
The [Teams docs](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/send-proactive-messages?tabs=dotnet#proactively-install-your-app-using-graph) (or a writer on that team) may be able to help. There was an old thread here a couple years ago, so I don't know if that information is still relevant, but I'll try to find it again.

**Comment by JonathanFingold** [comment]
This thread may be of use: https://github.com/MicrosoftDocs/bot-docs/issues/1687

**Comment by DonHarishAxe** [comment]
Thanks for the reply. I checked these threads but couldn't arrive at anything concrete. Can you please include someone from the team that may have an idea of this?

**Comment by JonathanFingold** [comment]
Someone from the support team should reach out soonish.

**Comment by ramfattah** [comment]
@DonHarishAxe 

As mentioned in [bug_report.md](https://github.com/MicrosoftDocs/bot-docs/blob/live/.github/ISSUE_TEMPLATE/bug_report.md), The Microsoft Bot Framework team prefers that **how to** questions be submitted on [Stack Overflow](http://stackoverflow.com/search?q=%23botframework). 
The official [Github](https://github.com/Microsoft/BotBuilder/blob/master/README.md) repos are the preferred platform for submitting **bug fixes** and **feature requests**.

**Comment by DonHarishAxe** [comment]
@ramfattah I had another question on bot framework that I created on stackoverflow - https://stackoverflow.com/questions/72917828/how-to-send-proactive-message-to-multiple-teams-group-chats. I have not received any response in about two weeks. Anyway, I have created a stackoverflow question for this too now - https://stackoverflow.com/questions/73073614/how-to-find-service-url-for-proactive-messages.
